{"uuid": "8aedb3bc-a802-4069-8943-f4dd3f8ca3f7", "children": ["619091f5-abae-4a0a-9118-d206afd0cf4b"], "befores": [{"name": "_UnitTestCase__pytest_class_setup", "status": "passed", "start": 1594376754161, "stop": 1594376778048}], "afters": [{"name": "_UnitTestCase__pytest_class_setup::0", "status": "broken", "statusDetails": {"message": "KeyboardInterrupt\n", "trace": "  File \"C:\\Users\\admin\\AppData\\Roaming\\Python\\Python38\\site-packages\\allure_commons\\_allure.py\", line 185, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"c:\\python38\\lib\\site-packages\\_pytest\\fixtures.py\", line 800, in _teardown_yield_fixture\n    next(it)\n  File \"c:\\python38\\lib\\site-packages\\_pytest\\unittest.py\", line 105, in fixture\n    teardown()\n  File \"E:\\LocalYHYH\\scripts\\test_CBJK.py\", line 22, in tearDownClass\n    GetDriver().quit_driver()\n  File \"E:\\LocalYHYH\\eles\\get_driver.py\", line 20, in quit_driver\n    cls.driver.quit()\n  File \"c:\\python38\\lib\\site-packages\\selenium\\webdriver\\chrome\\webdriver.py\", line 153, in quit\n    RemoteWebDriver.quit(self)\n  File \"c:\\python38\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 698, in quit\n    self.execute(Command.QUIT)\n  File \"c:\\python38\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 319, in execute\n    response = self.command_executor.execute(driver_command, params)\n  File \"c:\\python38\\lib\\site-packages\\selenium\\webdriver\\remote\\remote_connection.py\", line 374, in execute\n    return self._request(command_info[0], url, body=data)\n  File \"c:\\python38\\lib\\site-packages\\selenium\\webdriver\\remote\\remote_connection.py\", line 397, in _request\n    resp = self._conn.request(method, url, body=body, headers=headers)\n  File \"c:\\python38\\lib\\site-packages\\urllib3\\request.py\", line 75, in request\n    return self.request_encode_url(\n  File \"c:\\python38\\lib\\site-packages\\urllib3\\request.py\", line 97, in request_encode_url\n    return self.urlopen(method, url, **extra_kw)\n  File \"c:\\python38\\lib\\site-packages\\urllib3\\poolmanager.py\", line 336, in urlopen\n    response = conn.urlopen(method, u.request_uri, **kw)\n  File \"c:\\python38\\lib\\site-packages\\urllib3\\connectionpool.py\", line 670, in urlopen\n    httplib_response = self._make_request(\n  File \"c:\\python38\\lib\\site-packages\\urllib3\\connectionpool.py\", line 392, in _make_request\n    conn.request(method, url, **httplib_request_kw)\n  File \"c:\\python38\\lib\\http\\client.py\", line 1240, in request\n    self._send_request(method, url, body, headers, encode_chunked)\n  File \"c:\\python38\\lib\\http\\client.py\", line 1286, in _send_request\n    self.endheaders(body, encode_chunked=encode_chunked)\n  File \"c:\\python38\\lib\\http\\client.py\", line 1235, in endheaders\n    self._send_output(message_body, encode_chunked=encode_chunked)\n  File \"c:\\python38\\lib\\http\\client.py\", line 1006, in _send_output\n    self.send(msg)\n  File \"c:\\python38\\lib\\http\\client.py\", line 946, in send\n    self.connect()\n  File \"c:\\python38\\lib\\site-packages\\urllib3\\connection.py\", line 187, in connect\n    conn = self._new_conn()\n  File \"c:\\python38\\lib\\site-packages\\urllib3\\connection.py\", line 159, in _new_conn\n    conn = connection.create_connection(\n  File \"c:\\python38\\lib\\site-packages\\urllib3\\util\\connection.py\", line 74, in create_connection\n    sock.connect(sa)\n"}, "start": 1594376778779, "stop": 1594376784923}], "start": 1594376754161, "stop": 1594376784924}